pyblish_qml/models.py

```diff
@@ -384,7 +384,11 @@ def add_instance(self, instance):
         item["hasCompatible"] = True
 
         # Visualised in Perspective
-        item["familiesConcatenated"] = instance["data"].get("family", "")
```

Comment:
A little too verbose I think. How about something like this?

```python
", ".join(
  family for family in
  [instance["data"].get("family")] +
  instance["data"].get("families", [])
  if family is not None
)
```

Could maybe make a function out of it.

```python
def families(instance):
  return list(
    family for family in
    [instance["data"].get("family")] +
    instance["data"].get("families", [])
    if family is not None
  )

", ".join(families(instance))
```